be/src/olap/row.h

```diff
@@ -134,6 +134,19 @@ void copy_row(DstRowType* dst, const SrcRowType& src, MemPool* pool) {
     }
 }
 
+// NOTE: only use in MemTable for now, it is same with copy_row(), but for HLL and Object type,
+// its content is in Slice->data, other than a normal Slice, so we just assign the Slice->data
+// pointer, instead of memcpy the data.
+// TODO(lingbin): remove this method
+template<typename DstRowType, typename SrcRowType>
+void copy_row_in_memtable(DstRowType* dst, const SrcRowType& src, MemPool* pool) {
```

Comment:
If this is only used for MemTable, is the template necessary?

Reply:
Since this method not only needs to copy the HLL type, but also all other types. If templates are not used, then we need to implement a corresponding copy_object () for all types. Moreover, if all types are handled in one method, it means a bunch of switch-case statements. This way maybe more ugly.